Sources/Slacket/Kitura/Middleware/HealthCheckMiddleware.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  HealthCheckMiddleware.swift
+//  Slacket
+//
+//  Created by Jeremi Kaczmarczyk on 22.06.2016.
+//
+//
+
+import Foundation
+
+import Kitura
+import HeliumLogger
+import LoggerAPI
+
+class HealthCheckMiddleware: RouterMiddleware {
+    
+    func handle(request: RouterRequest, response: RouterResponse, next: () -> Void) throws {
+        try response.end()
```

Comment:
This may not be the case anymore but in one of the previous versions of Kitura you had to add status `.ok` manually. Have you checked that it actually returns 200?